content/html/HTML.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: "HTML: Hypertext Markup Language"
 mdn_url: /en-US/docs/Web/HTML
+spec_url: https://html.spec.whatwg.org/multipage/
```

Comment:
I hadn't thought of this, but having spec links on landing pages is really nice.